aws/rhel-8.9-nightly-aarch64/main.tf

```diff
@@ -0,0 +1,53 @@
+module "aws" {
+  source = "../_base"
+
+  name             = "rhel-8.9-nightly-aarch64"
+  ami              = "ami-0a0b540c0fdd4ff3e"
```

Comment:
Is the ami correct? It might have gotten removed by cloud cleaner, did you tag it with `persist: true` ?

Reply:
Obviously not :-(.